doc/api/buffer.md

```diff
@@ -175,8 +175,8 @@ string into a `Buffer` as decoding.
   encoding a `Buffer` to a string, this encoding will omit padding.
 
 * `'hex'`: Encode each byte as two hexadecimal characters. Data truncation
-  may occur when decoding strings that do exclusively contain valid hexadecimal
-  characters. See below for an example.
+  may occur when decoding strings that do not exclusively contain valid
+  hexadecimal characters. See below for an example.
```

Comment:
```suggestion
  may occur when decoding strings that do not exclusively consist of an even
  number of hexadecimal characters. See below for an example.
```

Maybe something like this?

Reply:
I think that this, plus another code example (something like `123` -> `12`) would be very nice

Reply:
@tniessen WDYT?

Reply:
SGTM. These are unfortunate oddities that we need to maintain for backward compatibility :(

Reply:
I took a better look at the examples and figured that I could achieve the same thing by altering the strings used in the examples.